015-exploratory-test-module.md

```diff
@@ -0,0 +1,163 @@
+# Exploratory test repository
+
+The exploratory test will contain two dialogue applications, which gives you the possibility to create your test scenarios. 
+
+## Motivation
+
+Every release, we do exploratory testing and test Strimzi. 
+This tool also provides `transcript` for all scenarios that the user tried. In other words, the application stores success and wrong scenarios. 
```

Comment:
It sounds a little bit weird....the `tool also do something` even when don't know what it's doing in basic function :)